mypyc/emit.py

```diff
@@ -166,23 +167,23 @@ def tuple_unique_id(self, rtuple: RTuple) -> str:
         in the same way python can just assign a Tuple[int, bool] to a Tuple[int, bool].
         """
         if rtuple not in self.context.tuple_ids:
-            self.context.tuple_ids[rtuple] = str(len(self.context.tuple_ids))
+            self.context.tuple_ids[rtuple] = rtuple.accept(TupleNameVisitor())
```

Comment:
Can we just remove this method and the `tuple_ids` dict? As far as I can see, we can just always calculate the name of a tuple struct without caching.

Reply:
@JukkaL If I create 10 different int-only tuples (of size 10, let's say), isn't it helpful to have the cache to avoid unnecessarily visiting 90 elements via TupleNameVisitor?